ROADMAP.md

```diff
@@ -0,0 +1,95 @@
+# Q2 2020 Roadmap
+
+Each quarter the team will highlight areas of focus for our work and upcoming research.
+ 
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](.github/CORE_SERVICES.md), and internal priorities. When community pull requests exist for a given item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp will complete any additional work needed. 
+
+Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. 
+
+To make contribution easier, we’ll be using the [`Help Wanted`](https://github.com/terraform-providers/terraform-provider-aws/labels/help%20wanted) tag to point to issues we’d like to include in this quarter’s series of releases. 
+
+This quarter (May-July ‘20) we will be prioritizing the following areas of work: 
+
+## Currently In Progress
+
+### 3.0.0 
+
+Milestone: [v3.0.0](https://github.com/terraform-providers/terraform-provider-aws/milestone/70)
+Issue: [#8950](https://github.com/terraform-providers/terraform-provider-aws/issues/8950)
```

Comment:
This issue only partially represents the 3.0.0 work -- is it worth listing more than this issue, creating a placeholder issue that summarizes everything, or dropping this?

Reply:
🤔 It might be worth dropping the issue or listing multiple? I'm open to either - dropping the issue seems to be the most straight forward.